internal/unionstore/art/art.go

```diff
@@ -392,51 +393,89 @@ func (t *ART) Size() int {
 }
 
 func (t *ART) checkpoint() arena.MemDBCheckpoint {
-	panic("unimplemented")
+	return t.allocator.vlogAllocator.Checkpoint()
 }
 
 func (t *ART) RevertNode(hdr *arena.MemdbVlogHdr) {
```

Comment:
Is "RevertVLogEntry" or something similar a more appropriate name? `RevertNode` looks like we are deleting the node.

Reply:
Rename it to `RevertVAddr`.